modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -569,7 +569,7 @@ export default class LargeVideoManager {
      */
     _updateVideoResolutionStatus() {
         const { height, width } = this.videoContainer.getStreamSize();
-        const isCurrentlyHD = Math.min(height, width) >= config.minHDSize;
```

Comment:
Isn't it good idea to also add a default value here? Just in case that this config prop is `undefined`.